web/next-env.d.ts

```diff
@@ -1,6 +1,3 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
-
-// NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
Looks like this removal was an accident?

Reply:
This issue appeared after running the 'ship-it.sh' script from my local machine; I didn't alter this particular file. I've created chore #298 to track this issue and added it to the backlog. I'll bring it up during today's stand-up.